spacy/ml/models/coref.py

```diff
@@ -0,0 +1,18 @@
+from typing import List
+from thinc.api import Model
+from thinc.types import Floats2d
+
+from ...util import registry
+from ...tokens import Doc
+
+
+@registry.architectures.register("spacy.Coref.v0")
```

Comment:
I think we normally use this as a callable, so just `registry.architectures`?

Reply:
I'm not sure I understand what you mean?

Reply:
Oh wait, you're right, like we'd do `@registry.loggers`. For some reason the architectures always use `.register` - I can clean that up on the `master` branch.